test_schemas/en/test_submit_page.json

```diff
@@ -4,17 +4,12 @@
     "schema_version": "0.0.1",
```

Comment:
The file should be named `test_submit_with_custom_submission_text.json` to be consistent with the equivalent schema that has a summary.